path.cabal

```diff
@@ -21,49 +21,49 @@ library
   hs-source-dirs:    src/
   ghc-options:       -Wall -O2
   exposed-modules:   Path, Path.Internal
-  build-depends:     base >= 4 && <5
-                   , exceptions
-                   , filepath < 1.2.0.1 || >= 1.3
-                   , template-haskell
+  build-depends:     aeson
+                   , base       >= 4.7     && < 5.0
                    , deepseq
-                   , aeson
-                   , hashable >= 1.2 && < 1.3
+                   , exceptions >= 0.6     && < 0.9
+                   , filepath   < 1.2.0.1  || >= 1.3
+                   , hashable   >= 1.2     && < 1.3
```

Comment:
Where do you take the lower bounds on exceptions and hashable from? If there's any doubt about these, I'd prefer to leave them off.